float32 roll
float32 pitch
float32 yaw
float32 height
float32 x_shift
float32 stance_x
float32 stance_y

int32 run_time